core/mono-samples/iOS/NuGet.config

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <config>
+    <add key="repositoryPath" value="/Users/mdhwang/core/mono-samples/iOS/" />
+  </config>
+  <packageSources>
+    <add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
```

Comment:
Is this URL OK? Not familiar with it..

Reply:
I believe it contains the mono runtime? @steveisok